Startup/FullTime/PunishmentDay.js

```diff
@@ -1,35 +1,65 @@
-{
-    if (!isVar(VARIABLE_NEXT_PUNISHMENT_DAY)) {
-        sendVirtualAssistantMessage("Today it's thursday");
-        sendVirtualAssistantMessage("Thursdays are special");
-        sendVirtualAssistantMessage("Thursdays are confession days");
-        sendVirtualAssistantMessage("Since you're full time it is required that you at least launch Tease-AI for a weekly punishment check up");
-        sendVirtualAssistantMessage("It's expected that you've brought your level of punishment points sufficiently low");
-        sendVirtualAssistantMessage("If you haven't I'm adding more");
-        sendVirtualAssistantMessage("A lot more!");
-    } else {
-        sendVirtualAssistantMessage("%SlaveName% it's thursday")
-    }
-
-    //Only continue if we haven't check this today
-    if (!isVar(VARIABLE_NEXT_PUNISHMENT_DAY) || getDate(VARIABLE_NEXT_PUNISHMENT_DAY).hasPassed()) {
-        if (!getDate(VARIABLE_NEXT_PUNISHMENT_DAY).sameDay(setDate())) {
-            sendVirtualAssistantMessage("You've been skipping punishment day %SlaveName%!");
-            sendVirtualAssistantMessage("I've added a healthy dose of punishment points for that!");
-            addPunishmentPoints(300);
-            sendVirtualAssistantMessage("You are expected to report on tuesdays!");
-        }
-
-        //Set the date to the next thursday
-        setDate(VARIABLE_NEXT_PUNISHMENT_DAY, setDate(VARIABLE_NEXT_PUNISHMENT_DAY).addDay(7).setHour(0).setMinute(0).setSecond(0));
-
-        sendVirtualAssistantMessage("You have " + getVar(VARIABLE_PUNISHMENT_POINTS) + " punishment points");
-
-        if (getVar(VARIABLE_PUNISHMENT_POINTS) >= 100) {
-            sendVirtualAssistantMessage("Which isn't good enough!");
-            sendVirtualAssistantMessage("So I'm adding more!");
-            sendVirtualAssistantMessage("Make sure you report for punishment!");
-            addPunishmentPoints(250);
-        }
-    }
+{
+    if (!isVar(VARIABLE_NEXT_PUNISHMENT_DAY)) {
+        sendVirtualAssistantMessage("Today it's thursday");
+        sendVirtualAssistantMessage("Thursdays are special");
+        sendVirtualAssistantMessage("Thursdays are punishment days");
+        sendVirtualAssistantMessage("Since you're full time it is required that you at least launch Tease-AI for a weekly punishment check up");
+        sendVirtualAssistantMessage("It's expected that you've brought your level of punishment points sufficiently low");
+        sendVirtualAssistantMessage("If you haven't I'm adding more");
+        sendVirtualAssistantMessage("A lot more!");
+		//setDate(VARIABLE_NEXT_PUNISHMENT_DAY, setDate());
+    } else {
+        sendVirtualAssistantMessage("%SlaveName% it's thursday")
+    }
+
+    //Only continue if we haven't check this today
+    if (!isVar(VARIABLE_NEXT_PUNISHMENT_DAY) || getDate(VARIABLE_NEXT_PUNISHMENT_DAY).hasPassed()) {
+		//clear out all the punishment reasons from last week

```

Comment:
As I said I will think of a better system for this

Reply:
you have already have alot to do, and the more you change, the less others are able to help. (i.e. without an image directory people can't help generate content)  I'd suggest this is a case of "done is better than perfect".